FV3/conf/configure.fv3.theia

```diff
@@ -40,6 +40,10 @@ CFLAGS := $(INCLUDE)
 
 FFLAGS := $(INCLUDE) -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -nowarn -sox -align array64byte
 
+ifeq ($(GT4PY_DEV),Y)
+CPPDEFS += -DGT4PY_DEV
+endif
+
```

Comment:
Do we really want this in every configure file?

Reply:
I think we could consider deleting the files we are not using/not supporting. We have no idea if they even work on those systems with the changes we made to get the code to compile in docker.

Reply:
Ok, you convinced me.  The duplication of the configuration files with slight tweaks in each makes me anxious haha, but that's the convention and probably the easiest to manage in this system! Not copying additional flags throughout just makes it worse, thanks for spotting and correcting this situation.

Reply:
I like the idea/view of only supporting a limited number of configuration files.  The Linux/gnu and Docker/gnu configure files should probably be amalgamated into a single file.  We will still need the two Cray/Daint configurations (GNU and Intel).